python/cudf/cudf/tests/general_functions/test_to_numeric.py

```
@@ -244,18 +242,16 @@ def test_to_numeric_downcast_string_large_float(data, downcast):
         pd.Series(["1", "a", "3", ""]),  # mix of unconvertible and empty str
     ],
 )
-@pytest.mark.parametrize("errors", ["ignore", "raise", "coerce"])
+@pytest.mark.parametrize("errors", ["raise", "coerce"])
 def test_to_numeric_error(data, errors):
     if errors == "raise":
         with pytest.raises(
             ValueError, match="Unable to convert some strings to numerics."
```

Ruff (0.15.12)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)